torch/_tensor_docs.py

```diff
@@ -3576,3 +3576,12 @@ def callable(a, b) -> number
 If ``n`` is the number of dimensions in ``x``,
 ``x.T`` is equivalent to ``x.permute(n-1, n-2, ..., 0)``.
 """)
+
+add_docstr_all('as_subclass',
+               r"""
+as_subclass(cls) -> Tensor
+
+Makes a ``cls`` instance with the same data pointer as ``self``. Changes
+in the output mirror changes in ``data``. ``cls`` must be a subclass of
```

Comment:
Not sure what do you mean by `data` here? Does it reference the `.data` field of Tensors?
Isn't this closer to the `.detach()` semantic?

Reply:
Should have been `self`. It stays connected to the autograd graph.

Reply:
Please read the newer version to see if it's clearer.

Reply:
Looks good thanks !